wazo_confgend/confgen.py

```diff
@@ -21,13 +19,37 @@
 from wazo_confgend.template import new_template_helper
 from xivo_dao.helpers.db_utils import session_scope
 from twisted.internet.protocol import Protocol, ServerFactory
+import twisted.python.failure
 
 logger = logging.getLogger(__name__)
 
 
 class Confgen(Protocol):
+    def connectionMade(self):
+        logger.info("connection established")
+
+    def connectionLost(self, reason: twisted.python.failure.Failure):
```

Comment:
If this is importable and has no conflicts it might be cleaner to import it and type as `reason: Failure`